docs/infra/docker-composer-to-k8s-manifest.md

```diff
@@ -8,6 +8,10 @@ automatically convert our `docker-compose.yml` into a `k8s-maniftest.yml`.
 Ultimately the work did not succeed and we are instead manually migrating the
 orchestration into k8s.
 
+## Manual Migration
+
+The documentation for this approach is in a [separate document](k8s-migration.md).
```

Comment:
I prefer fully qualified urls, `[separate document](/docs/infra/k8s-migration.md)` this is mostly based on making it easier to globally search and replace when moving files around (and strongly influenced on react projects where it's dot-dot soup trying to figure out how to "find" the right reference).

Reply:
I also prefer link text that is more description.

```
Documentation on our approach to [manual k8s manual migrations](/docs/infra/k8s-migration.md)
```